pkg/kubelet/types/constants.go

```diff
@@ -17,16 +17,22 @@ limitations under the License.
 package types
 
 const (
-	// system default DNS resolver configuration
+	// ResolvConfDefault represents system default DNS resolver configuration
 	ResolvConfDefault = "/etc/resolv.conf"
 
-	// different container runtimes
+	// DockerContainerRuntime represent docker runtime
 	DockerContainerRuntime = "docker"
+	// RemoteContainerRuntime represent remote runtime
 	RemoteContainerRuntime = "remote"
 
-	// User visible keys for managing node allocatable enforcement on the node.
+	// Following is user visible keys for managing node allocatable enforcement on the node.
+
+	// NodeAllocatableEnforcementKey represents key for pods allocatable
```

Comment:
Use more descriptive comments. See https://github.com/kubernetes/community/blob/master/contributors/design-proposals/node/node-allocatable.md#phase-2---enforce-allocatable-on-pods

Reply:
thanks for your review. I will have more descriptions.